src/components/AuthorView.jsx

```diff
@@ -74,7 +75,14 @@ class AuthorView extends Component {
               <h1 className='author-view__header' onDoubleClick={this.handleNameClick}>{this.props.author}</h1>
             )
           }
-          <h2 id='author-id'>{this.props.currentAuthorId || this.props.me}</h2>
+          {!isId(this.props.author) && this.props.currentAuthorId === this.props.author
```

Comment:
can you explain this line? i don't understand the `currentAuthorId === author` part

Reply:
Basically if a user searches for someone that doesn't exist we originally show what they typed both as the user name and as the id. Instead, I decided to check to make sure authorId and author weren't the same thing (meaning a non existent user was searched for and has no actual id) and then display some text to let people know that the user was not found vs. doubling what they searched.

In the case that what they type is a valid id, gester automatically shows the username and id so I didn't need to handle that. But, I am checking if author is an Id in the event they click on someone who hasn't loaded properly and gester is showing their id as their username. I didn't want that to come up as 'user not found'.

Let me know if you need any more clarification.